src/main/kotlin/mate/academy/LetterRemover.kt

```diff
@@ -1,5 +1,19 @@
 package mate.academy
 
+//object LetterRemover {
+//    @JvmStatic
+//    fun removeLetter(input: String): CharArray {
+//        val replacedString = input.lowercase().replace('r', '!')
+//        return replacedString.toCharArray()
+//    }
+//}
+
 class LetterRemover {
-    // implement ...
+    companion object {
+        @JvmStatic
+        fun removeLetter(input: String): CharArray {
```

Comment:
FYI
You may simplify similar functions syntax in the future using kotlin possibilities.
Something like this:
fun removeLetter(input: String) = input.lowercase().replace('r', '!').toCharArray()